docs/contributing/how-to-contribute-to-docs.md

```diff
@@ -94,8 +94,22 @@ Every PR is building website and on success it shows the link to preview.
 
 We use [Netlify](https://www.netlify.com/) for hosting. We are using Open Source license (PRO). Thanks Netlify for this!
 
-On every commit to `master` netlify runs CI that invokes `make web` (defined in [netlify.toml](https://github.com/improbable-eng/thanos/blob/master/netlify.toml)))
+On every commit to `master` netlify runs CI that invokes `make web` (defined in [netlify.toml](/netlify.toml))
 
 NOTE: Check for status badge in README for build status on the page.
 
 If master build for netlify succeed, the new content is published automatically.
+
+## Test locally
+
+To test any changes to the docs locally, just run the following command to build the documention website:
+
+```bash
+make web
+```
+
+To start serving the website locally run the following command and you will be able to access the website on `localhost:1313` by default:
```

Comment:
Worth to add that you don't need to run `make web` if you do `make web-serve`? Easy to be confused by reading this and think that those two steps are essential, not sure.

Reply:
Yeah, that makes sense, sorry docs are hard to write :D I rephrased it a bit, just kept the `make web-serve` as the other one is explained further down anyways.